### ISSUE TITLE
Zoom does not Update Crosshairs in MPR Mode

### ISSUE THREAD

**Issue by matthiasg** [OHIF/Viewers#1419]
### Describe the Bug

Crosshairs do not update during zooming/scaling of the image. After a zoom the positions are not updated which can mislead a physician to wrongly locate something.

### What steps can we follow to reproduce the bug?

Open

https://viewer.ohif.org/viewer/1.3.6.1.4.1.25403.52237031786.3872.20100510032220.1

Wait for MPR reconstruction
Move crosshairs
Zoom in one of the regions
Only after clicking on one of the other images again it will jump correctly again

![Viewer-Crosshair-Zoom-Error](https://user-images.githubusercontent.com/231889/73819826-aee22200-47f0-11ea-9061-3c24e6a92748.gif)

**Comment by dannyrb** [comment]
I believe this is something @JamesAPetts has experiencing with. He may be able to offer guidance, or speak to our progress on this issue.

**Comment by JamesAPetts** [comment]
This is fixed in `react-vtkjs-viewport` 0.7.2, and will soon be in OHIF, included in PR #1392 :).

**Comment by fedorov** [comment]
Ok, so crosshairs update, but, unless I am missing something, they do not update properly.

In the animation below, the horizontal line corresponds to the location of the axial slice, which does not change with the zoom, so that line should stay in the same place relative to the patient. Right? Instead, it moves up and down.

![2020-07-09_22-52-36](https://user-images.githubusercontent.com/313942/87111369-336d1c80-c237-11ea-944c-558d6692eae2.gif)

**Comment by JamesAPetts** [comment]
Hey @fedorov, thanks for reviving this. Yeah this is a known issue. So the original fix here worked as long as the canvas doesn't change size. In your demo gif, a side panel had been opened/closed during the lifecycle of the viewport, as I can see your reference lines don't reach all the way to the top of the viewport.

Additionally the lines aren't _really_ implemented as reference lines, @swederik  just put horizontal lines in as a draft when the viewport was being made for OHIF back when OHIF switched to react. 

To summarise, 2 large known issues remain:
- The viewport does not react to css resizing properly.
- If you rotate a viewport the lines do not rotate, but moving the focus will still correctly recentre the other viewports.

I've wanted to address these issues for some time, but unfortunately VTK/MPR has been low priority for OHIF for a while now, that will likely change again in the future, but I'm not aware of when.

Could this potentially be an IDC issue?

**Comment by fedorov** [comment]
@JamesAPetts, thanks - we can definitely add as IDC candidate.

**Comment by fedorov** [comment]
This is still an issue - reproducible when changing the width of the browser window - crosshair location is changing in axial, but stays in place for other planes.

**Comment by matthiasg** [comment]
@fedorov may I ask what IDC stands for ? The readme commit references IDC funding but IDC does not seem to be mentioned inside the README.

**Comment by matthiasg** [comment]
As a follow up to this, we are in the process of developing a fully approved viewer on this and working through the risk assessment we defined a number of automated regression test scenarios which would need to be developed. Is there a framework in place that we could or should use to increase the chances of pushing it back into this main repo ?

**Comment by fedorov** [comment]
> @fedorov may I ask what IDC stands for ?

@matthiasg this PR should explain the connection with IDC: https://github.com/OHIF/Viewers/pull/2255/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R264.

**Comment by matthiasg** [comment]
Thanks.

**Comment by Punzo** [comment]
not reproducible anymore with lastest
